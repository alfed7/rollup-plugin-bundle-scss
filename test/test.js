const fs = require('fs').promises;
const path = require('path');
const util = require('util');
const exec = util.promisify(require('child_process').exec);

function resolve(...args) {
  return path.resolve(__dirname, ...args);
}

async function run(name) {
  await exec(`rollup -c=${resolve('cases', name, 'rollup.config.js')}`);
  const scss = await fs.readFile(resolve('cases', name, './dist/index.scss'), 'utf-8');
  expect(scss).toMatchSnapshot();
}

describe('cases', () => {
  test('basic', () => run('basic'));
  test('vue', () => run('vue'));
});
