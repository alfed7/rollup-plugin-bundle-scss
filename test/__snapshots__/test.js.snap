// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cases basic 1`] = `
".bar {
  color: #bf4080;
}


.foo {
  color: #cf649a;
}
"
`;

exports[`cases vue 1`] = `
"
.foo {
  color: #41b883;
}


.bar {
  color: #35495d;
}
"
`;
